chapters/ch30.md

```diff
@@ -1,51 +1,51 @@
 > # 30. Interpreting learning curves: High bias
 
---> # 30. Hiểu rõ [thông hiểu/giải thích] đồ thị quá trình học: Độ chệch cao
+# 30. Hiểu rõ [thông hiểu/giải thích] đồ thị quá trình học: Độ chệch cao
 
 > Suppose your dev error curve looks like this:
 
---> Giả sử đường cong sai số của tập phát triển thu được hình sau:
+Giả sử đường cong sai số của tập phát triển thu được hình sau:
 
 ![img](../imgs/C30_01.png)
 
 > We previously said that, if your dev error curve plateaus, you are unlikely to achieve the desired performance just by adding data.
 
---> Như ta đã thảo luận, nếu đường cong sai số của tập phát triển trải dài và bằng phẳng [cần tìm từ khác?], việc chỉ thêm dữ liệu sẽ không thể đem về hiệu suất mà ta mong đợi. 
+Như ta đã thảo luận, nếu đường cong sai số của tập phát triển trải dài và bằng phẳng [cần tìm từ khác?], việc chỉ thêm dữ liệu sẽ không thể đem về hiệu suất mà ta mong đợi. 
 
 > But it is hard to know exactly what an extrapolation of the red dev error curve will look like. If the dev set was small, you would be even less certain because the curves could be noisy.
 
---> Tuy nhiên, sẽ thật khó để dự đoán chính xác đường cong màu đỏ thể hiện sai số của tập phát triển sẽ trông như thế nào. Trong trường hợp tập phát triển nhỏ, việc dự đoán chính xác sẽ càng trở nên khó khăn vì khi đó đường cong này sẽ có khả năng bị nhiễu. 
+Tuy nhiên, sẽ thật khó để dự đoán chính xác đường cong màu đỏ thể hiện sai số của tập phát triển sẽ trông như thế nào. Trong trường hợp tập phát triển nhỏ, việc dự đoán chính xác sẽ càng trở nên khó khăn vì khi đó đường cong này sẽ có khả năng bị nhiễu. 
```

Comment:
> đường cong này sẽ có khả năng bị nhiễu

"curves could be noisy" tức là thế nào bạn nhỉ? mình đang dịch chap 32, cũng gặp 1 cụm tương tự "curve looks slightly noisy"; hiện chưa biết dịch ntn vì chưa rõ nghĩa của nó.

Reply:
@william-vu Ở phần của em thì em hiểu là với 1 tập phát triển nhỏ thì sẽ khó có thể dự đoán xu hướng của đường cong đó khi mà thêm dữ liệu vào tập phát triển. Nên noisy curve ở đây có thể chỉ một đường cong không có 1 xu hướng nhất định (lúc lên, lúc xuống) để dự đoán được hình dáng của nó khi mà thêm dữ liệu vào. Em cũng không biết còn từ nào không vì từ nhiễu có vẻ không đúng lắm.

Em đọc qua chương 32 có vẻ từ noisy cũng có nghĩa như thế thì phải. Ở trên chắc tác giả ám chỉ những giá trị ở trong curve cao hơn, hoặc nhỏ hơn xu hướng chung.

Them em hiểu là  như thế, nhưng mà em cũng có thể sai. Có lẽ đợi mn cho ý kiến để dịch phần này mượt hơn.

Reply:
@khoapip: chap 32 a thấy tác giả có câu này nữa "Having a small training set means that the dev and training errors may randomly fluctuate.", chắc noisy = randomly fluctuate.